packages/dataviews/src/dataviews.tsx

```diff
@@ -42,7 +48,7 @@ type DataViewsProps< Item > = {
 		totalItems: number;
 		totalPages: number;
 	};
-	supportedLayouts: string[];
+	defaultLayouts: SupportedLayouts;
```

Comment:
Not from here, but probably better to move all types to the respective file.

Reply:
which respective file?

Reply:
`types.ts`

Reply:
I prefer for the `types.ts` to stay mostly about the structural types of the package and the temporary types that are specific to components... I prefer to keep them next to the components directly. These types don't have any value as shared types.